ckb-mercury-sdk/src/main/java/org/nervos/mercury/model/req/ToInfo.java

```diff
@@ -4,9 +4,9 @@
 
 public class ToInfo {
   public String address;
-  public BigInteger amount;
+  public long amount;
```

Comment:
Also can be UDT amount here.

Reply:
So far mercury change is not ready for test. I plan to refactor this part after mercury change so just leave them here now.

Reply:
ok